Develop

PythonZem opened the pull request.

PythonZem committed:
add format_linter_error function

PythonZem committed:
add format_single_linter_file

PythonZem committed:
add some operators to velue (key:  error / status) format_single_linter_file

PythonZem committed:
solition 1

PythonZem committed:
fix flake8

PythonZem committed:
fix flake8 1

Serhii-Leonenko commented on app/main.py:
```diff
@@ -1,13 +1,20 @@
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {"errors": [] if len(errors) == 0
```
this check is redundant here

PythonZem replied:
remove this check

Serhii-Leonenko commented on app/main.py:
```diff
@@ -1,13 +1,20 @@
+    return {"errors": [] if len(errors) == 0
+            else [format_linter_error(error=error) for error in errors],
+            "path": file_path,
+            "status": "passed" if errors == [] else "failed"}
```
you should use one style for curly braces in your project

PythonZem replied:
Fixed this))

PythonZem committed:
implement last chenges

-  remuve rudemet check in format_single_linter_file
- write all the brackets in the same style

PythonZem committed:
fix my style for flake8

Serhii-Leonenko commented on app/main.py:
```diff
@@ -1,13 +1,20 @@
+            "message": error["text"],
+            "name": error["code"],
+            "source": "flake8"
+            }
```
Read the checklist, it's obligatory. In all functions you violate it.

PythonZem replied:
I re-read the checklist. I tried to fix all the brackets in one style

PythonZem committed:
Corrected braces

Serhii-Leonenko reviewed (approved):
well done!
